jsonargparse_tests/test_util.py

```diff
@@ -137,6 +137,8 @@ def test_path_equality_operator(paths):
 
 
 def test_path_file_access_mode(paths):
+    if is_posix and os.geteuid() == 0:
+        pytest.skip("User is root, skipping permission tests")
```

Comment:
Please skip using a decorator, similar to https://github.com/omni-us/jsonargparse/blob/1bab38ae7a5fe281c62fe784fc620bbbf259f2e8/jsonargparse_tests/conftest.py#L35-L38